src/app/common/components/dataset-kind/dataset-kind.component.html

```diff
@@ -0,0 +1,3 @@
+<span data-test-id="dataset-kind" class="label label--secondary v-align-middle badge-background">{{
```

Comment:
Can we make the color of this badge orange for root datasets and blue for derivative, just like in lineage graph:
<img width="400" alt="image" src="https://github.com/user-attachments/assets/07a29938-6ebd-49ae-807a-335fd7cb2620" />

I'd like to use the same colors throughout the system to create strong visual associations.